src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/PackagingTest.cs

```diff
@@ -397,7 +397,7 @@ public void CheckSignApk ([Values(true, false)] bool useApkSigner, [Values(true,
 			} else {
 				proj.SetAndroidSupportedAbis ("armeabi-v7a", "x86");
 			}
-			using (var b = CreateApkBuilder (Path.Combine ("temp", TestContext.CurrentContext.Test.Name))) {
+			using (var b = CreateApkBuilder (Path.Combine ("temp", TestName))) {
```

Comment:
I found this actually breaks the test, because it cleans this directory before generating the project:

```
(_Sign target) -> 
/Users/runner/work/1/s/bin/Release/dotnet/packs/Microsoft.Android.Sdk.Darwin/35.99.0-ci.pr.gh9664.93/tools/Xamarin.Android.Common.targets(2492,2): error XA4310: `$(AndroidSigningKeyStore)` file `/Users/runner/work/1/a/TestRelease/01-09_16.16.57/temp/CheckSignApkTrueTrue/release.keystore` could not be found.
```

Something like this might work?
* https://github.com/dotnet/android/pull/9395/commits/056595a8a749e92d72c89de68f8ffeefad166eb5